ivy/functional/frontends/jax/numpy/indexing.py

```diff
@@ -30,3 +30,9 @@ def take_along_axis(arr, indices, axis, mode="fill"):
 @to_ivy_arrays_and_back
 def tril_indices(n_rows, n_cols=None, k=0):
     return ivy.tril_indices(n_rows, n_cols, k)
+
+
+# indices
+@to_ivy_arrays_and_back
+def indices(dimensions, dtype=<class 'jax.numpy.int32'>, sparse=False):
+    return ivy.indices(dimensions, dtype, sparse)
```

Comment:
Are you sure you ran the tests locally and verified them? `ivy` does not have any methods named `indices`. If you run the tests, it will throw you this error right away. In this scenario, you will have to come up with a manual implementation of `indices` using just `ivy` methods. You can look into `jax`'s source code for reference [here](https://jax.readthedocs.io/en/latest/_modules/jax/_src/numpy/lax_numpy.html#indices) or go through Ivy's [docs](https://lets-unify.ai/ivy/) to look for the closest function in any other frontends or the functional API to what you're trying to implement. 🙂